### DISCUSSION TITLE
Teleport-cluster helm chart is supported by IBM cloud object storage

### DISCUSSION THREAD

**Discussion by PallaviPalisetti** [gravitational/teleport#10629] [Show and tell]
Hii there, 
 I'm curious if the teleport-cluster helm chart is compatible with IBM Cloud Object Storage for session recordings.

**Comment by webvictim** [comment]
The `teleport-cluster` Helm chart itself doesn't have a native IBM Cloud mode which will configure this, but if you set it up with `chartMode: custom` and put the relevant IBM COS config under `audit_sessions_uri` (see the [IBM Cloud docs](https://goteleport.com/docs/setup/deployments/ibm/#storage-ibm-cloud-file-storage)) in the Teleport config file then it will work.

You would also need to mount the IBM HMAC key to the `~/.aws/credentials` path inside the chart container, probably by creating a Kubernetes secret and then using `extraVolumes:` and `extraVolumeMounts:` to mount the secret in the correct place.

**Reply by PallaviPalisetti** [reply]
thanks for quick response.

**Reply by PallaviPalisetti** [reply]
Teleport is set up as a bastion host with a dns entry for the public IP address. M y architecture includes a k8s cluster, I must use either the teleport-cluster or kube-agent helm charts to connect to the bastion host for k8s access and applications hosted on the k8s .
